expenser-server/handler/handleTransaction.go

```
@@ -133,7 +156,7 @@ func HandleTransactionDelete(data model.Config) http.HandlerFunc {
 			return
 		}
 
-		data.DeleteTransactionFromDB(id)
+		data.DeleteTransactionFromDB(id, userID)
```

golangci-lint

Error return value of `data.DeleteTransactionFromDB` is not checked

(errcheck)